frontend-next-migration/src/features/NavigateHeroes/ui/HeroMenuAsDropdown.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useClientTranslation } from '@/shared/i18n';
+import { HeroSlug } from '@/entities/Hero';
+import HeroMenuAsDropdown from '@/features/NavigateHeroes/ui/HeroMenuAsDropdown';
```

Comment:
instead "import HeroMenuAsDropdown from '@/features/NavigateHeroes/ui/HeroMenuAsDropdown';"  should be "./HeroMenuAsDropdown"